src/app/home/home.component.ts

```diff
@@ -412,17 +474,26 @@ export class HomeComponent implements AfterViewInit {
         this.hasStoppedTypingForEventualMarking = false;
     }
 
-
+    /**
+     * Marks the editor based on the lastly (time-wise) typed character. For example breaking the current line is
```

Comment:
It doesn't seem like it **marks the editor**. Comment is a bit misleading. I would expect this comment in some **markEditor()** function.  
More sth like this or similiar: _Checks if the given emitted event key is included in a list of key triggers. The keyboard press of this triggers only should mark the editor._

Reply:
Made some changes, let me know if anything should be changed